code/drasil-website/lib/Drasil/Website/Body.hs

```diff
@@ -160,6 +161,15 @@ paperGOOL :: Reference
 paperGOOL = makeURI "GOOLPaper" (gitHubInfoURL ++ "/blob/master/Papers/GOOL/GOOL.pdf") (shortname' $ S "GOOLPaper")
 papersWiki :: Reference
 papersWiki = makeURI "papersWiki" (gitHubInfoURL ++ "/wiki/Drasil-Papers-and-Documents") (shortname' $ S "papersWiki")
+oldPaperWiki :: Reference
+oldPaperWiki = makeURI "oldPaperWiki" (danContributionPath
+  ++ "/ICSE%20Workshop%20-%20SE4Science/ICSE_LiterateFrameworkForSCSoftware_LSS.pdf") (shortname' $ S "oldPaperWiki")
+posterWiki :: Reference
+posterWiki = makeURI "posterWiki" (danContributionPath
+  ++ "/CAS%20Poster%20Competition/Poster/DrasilPoster.pdf") (shortname' $ S "posterWiki")
+newPaperWiki :: Reference
+newPaperWiki = makeURI "newPaperWiki" (gitHubInfoURL 
```

Comment:
By "Wiki", we usually either mean wikipedia or our own [wiki](https://github.com/JacquesCarette/Drasil/wiki). I think you can improve the names of these variables: icsePositionPaper, danPoster, and wellUnderstoodPaper are all decent names in line with the others in this file.